src/Components/Results/Programs/Programs.tsx

```diff
@@ -24,9 +24,7 @@ const Programs = () => {
       {Object.entries(categories).map(([category, programs]) => {
         return (
           <div key={category}>
-            <h2>
-              <ResultsTranslate translation={programs[0].category} />
-            </h2>
+            <CategoryHeading headingType={programs[0].category} amount="200"></CategoryHeading>
```

Comment:
Amount is hard coded. @Mdemenko1 created a function to get the value by category in her pr.